src/renderer/components/_molecules/data-types/structure/table/elements/array-modal.tsx

```
@@ -0,0 +1,284 @@
+                />
+                <SelectContent
+                  position='popper'
+                  side='bottom'
+                  sideOffset={-28}
+                  className='box z-[999] h-fit w-[--radix-select-trigger-width] overflow-hidden rounded-lg bg-white outline-none dark:bg-neutral-950'
+                >
+                  {types.map((type) => (
+                    <SelectItem
+                      value={type}
+                      className='flex w-full cursor-pointer items-center justify-center py-1 outline-none hover:bg-neutral-100 dark:hover:bg-neutral-800'
+                    >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)